docs/partner_editable/overview_target_and_usage.adoc

```diff
@@ -4,10 +4,11 @@
 
 This Quick Start reference deployment guide provides step-by-step instructions for deploying Rafay’s Kubernetes Operations Platform (KOP) for Amazon Elastic Kubernetes Service (Amazon EKS) management and operations. KOP lets you deploy, operate, and manage the lifecycle of Kubernetes clusters and containerized applications in both public-cloud and on-premises environments.
 
-This Quick Start is for IT and DevOps professionals who want to deploy Kubernetes clusters in the AWS Cloud, in data centers, and at network edges. KOP uses a software as a service (SaaS) management console that automates multicluster provisioning, configuration management, and upgrades. It also automates the deployment of containerized applications and enables access to Kubernetes clusters through a zero-trust connectivity model.
+With the Rafay Kubernetes Operations Platform, enterprises leverage a single operations platform for the lifecycle management of Amazon EKS clusters and containerized apps. Rafay empowers Platform and SRE  teams to carry out their function easily, resulting in faster deployments of new apps to production, reduced app downtimes, and elimination of security & compliance risk associated with their infrastructure.
```

Comment:
```suggestion
With the Rafay Kubernetes Operations Platform, enterprises use a single operations platform to manage the lifecycle of Amazon EKS clusters and containerized applications. With KOP, your platform and site-reliability engineering (SRE) teams can speed up the deployment of new applications to production, reduce application downtimes, and reduce security and compliance risks associated with their infrastructure.
```

Reply:
@troy-ameigh Also, per AWS style guidelines, please edit this sentence in line 5: 

From this ... "KOP lets you deploy, operate, and manage the lifecycle of Kubernetes clusters and containerized applications in both public-cloud and on-premises environments."

To this ... "With KOP, you can deploy, operate, and manage the lifecycle of Kubernetes clusters and containerized applications in both AWS Cloud and on-premises environments."